data/human/human missions.txt

```diff
@@ -6425,7 +6425,7 @@ mission "Care Package to South 2b"
 mission "Moving House 1"
 	minor
 	name "Moving House"
-	description "An anarchist named Kris has asked you to help them move house from <planet stopovers> to <destination>."
+	description "An anarchist named Kris has asked you to help them and <cargo> move house from <planet stopovers> to <destination>. Payment is <payment>."
```

Comment:
```suggestion
	description "An anarchist named Kris has asked you to help them move house with <cargo> from <planet stopovers> to <destination>. Payment is <payment>."
```

The whole "move house" phrasing is awkward to me, but I think this way around is less so?